crwatcher/watcher.go

```diff
@@ -152,6 +177,22 @@ func (cw *CRWatcher) setupController() {
 	)
 }
 
+// noFilterOrPassesFiltering checks to see if we are using an opt in filter (if not, then return true), and if so returns whether we
```

Comment:
This name is a lot longer than it was in the previous version, and starts to hit my internal thought of it's against go's mentality of short names. I can be convinced to leave it with any bit of objection against the shorter name.

Reply:
Shortened